browser/components/MarkdownEditor.js

```diff
@@ -142,6 +142,12 @@ class MarkdownEditor extends React.Component {
     this.renderPreview(this.props.value)
   }
 
+  handleKeyDown(e) {
+    if (this.state.status === 'CODE' && e.key == 'Escape') {
```

Comment:
No, it's not same. Yes, escape from editing md. By default I set `ESC` and I try to implement customizable key map.